abcclassroom/tests/test_assignment_template.py

```diff
@@ -132,7 +132,7 @@ def test_create_template_dir_move_git_dir(course_structure_assignment):
     # create a file in the template dir, init a git repo and commit
     testfile = Path(template_path, "file1.txt")
     testfile.touch()
-    github.init_and_commit(template_path, False)
+    abcgit.init_and_commit(template_path, False)
```

Comment:
@kcranston above we are importing as git vs abcgit . i kind of like abcgit so it's not confused with the git command itself. what do you think about importing it that way everywhere?

Reply:
Sure! I love consistency.